### PR TITLE
app: output updates

### PR DESCRIPTION
**Pull request by lolzballs**

### CODE REVIEW

File: app/growth_chart.py [hunk 1]
```diff
@@ -56,24 +73,45 @@ def _plot_thread(self):
...
-            plt.ylabel('Bone Age [months]')
-            plt.title('Bone Age Growth Chart')
+            plt.plot(CHRONOLOGICAL_AGE[input.sex],
+                     BOUND_LOW[input.sex],
+                     color = 'b', marker='o', markersize=14, linestyle = '--',
+                     label = 'Brush Foundation Study - lower bound')
+            plt.plot(CHRONOLOGICAL_AGE[input.sex],
+                     BOUND_HIGH[input.sex],
+                     color = 'b', marker='o', markersize=14, linestyle = '--',
+                     label = 'Brush Foundation Study- higher bound')
```
Comment (lolzballs) [inline app/growth_chart.py]: @katarinachiam Should the label be 'Brush Foundation Study - upper bound' instead?

Comment (katarinachiam) [reply app/growth_chart.py]: ya i'll change! thanks for catching that

File: app/growth_chart.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def _plot_thread(self):
             plt.plot(CHRONOLOGICAL_AGE[input.sex],
                      BOUND_HIGH[input.sex],
                      color = 'b', marker='o', markersize=14, linestyle = '--',
-                     label = 'Brush Foundation Study- higher bound')
+                     label = 'Brush Foundation Study- upper bound')
```
Comment (lolzballs) [inline app/growth_chart.py]: also the space before the dash?

Comment (katarinachiam) [reply app/growth_chart.py]: oui

Comment (lolzballs) [reply app/growth_chart.py]: ty, i'll squash the stuff together

**Pull request merged**